build/tools/releaser-post.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+# This script will automate the file alteration, which involves
+# 1. Updating version of the CLI in version/version.go
+# 2. Updating version of noobaa core image in pkg/options.go => ContainerImageTag
+# 3. Run make gen
+# 4. Commit the changes with automated commit message
+
```

Comment:
We might want to add PS4 for this script to see the debugging:
```suggestion

export PS4='\e[36m+ ${FUNCNAME:-main}@${BASH_SOURCE}:${LINENO} \e[0m'
```